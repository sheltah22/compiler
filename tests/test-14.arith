let x : int = (4 < 3) in 3 * x
