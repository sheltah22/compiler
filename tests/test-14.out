Fatal error: exception Failure("Let typechecking failed, expected binding type: int, actual: bool")
